src/index.ts

```diff
@@ -89,3 +110,16 @@ export function geometryToCells(geometry, resolution: bigint): Quadbin[] {
     max_zoom: zoom
   }).map(([x, y, z]) => tileToCell({x, y, z}));
 }
+
+export function quadbinToBoundary(cell: bigint) {
```

Comment:
The naming should be consistent with https://github.com/CartoDB/quadbin-py, so in this case `cellToBoundary`

Reply:
Sure!